src/stores/useAuthStore.ts

```diff
@@ -26,7 +26,7 @@ export const useUserStore = create<UserStoreState>()(
       },
     }),
     {
-      name: "feedbackUserStore",
+      name: "feedbackAuthStore",
```

Comment:
I'd name it `AuthStore` or `useAuthStore`

```suggestion
      name: "AuthStore",
```